doc/Installation/Install-LibreNMS.md

```diff
@@ -14,7 +14,7 @@ Connect to the server command line and follow the instructions below.
 > at `mysql>` prompts) or temporarily become a user with root
 > privileges with `sudo -s` or `sudo -i`.
 
-**Please note the minimum supported PHP version is 7.2.5**
+**Please note currently the minimum supported PHP version is 7.2.5, but from 1 November 2020 this will be bumped to PHP 7.3**
```

Comment:
that's in 4 weeks, why not directly increase to 7.3 ? :-)

Reply:
To lessen the confusion in Discord. New users will come and say they have installed minimum 7.3 and possibly confuse existing users who have not upgraded yet, causing mass panic. Just my thought.

Reply:
1. November means next release, so i think it could be changed now directly to it.
Panic is relative, with next release all below 7.3 get a Version warning ...

Reply:
FYI, I think the change will be the END of November. Which we call the November release ;)

Reply:
don't think it's such bad if people (have to) install newer version than actually required.
also this avoids an enforced PHP Upgrade for new Users, which means better user experience

Reply:
PR already updated accordingly.